recipes/neo4j-python-driver/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name='neo4j-python-driver' %}
-{% set version='1.1.0rc1' %}
```

Comment:
You can keep this version for now and then after this recipe is merged, you can upgrade to the latest version. At conda-forge we don't have different versions in subdirectories. We just update the recipe for new versions.